tests/libcxx/expected_results.txt

```diff
@@ -234,8 +234,10 @@ std/utilities/meta/meta.unary/meta.unary.prop/is_literal_type.deprecated.fail.cp
 
 # *** MISSING STL FEATURES ***
 # P2321R2 zip
-std/language.support/support.limits/support.limits.general/tuple.version.pass.cpp FAIL
-std/language.support/support.limits/support.limits.general/utility.version.pass.cpp FAIL
+# NOTE: We skip these rather than FAILing them because the __cpp_lib_ranges_zip feature test macro is defined

```

Comment:
Yeah, these should pass if we define the feature-test macro. That said, we shouldn't define the feature-test macro if we don't implement everything from the proposal that adds the feature-test macro, and IIRC we don't yet implement `zip_transform`?

Reply:
Nor `adjacent`

Reply:
This was done as a part of some earlier suggestions (see [this comment](https://github.com/microsoft/STL/pull/3035#issuecomment-1216942634)). I set those tests to skipped because they were defined as part of the libc++ test suite, and they failed because they do not define the feature-test macro. The way I saw it, there were two options: either don't define the feature-test macro or skip the tests.

As you can see, I chose the latter option, but that seems to be incorrect in hindsight. I'll need to reverse those changes.

Reply:
Conventionally we only define a feature-test macro to the value it was given by some proposal when we've implemented all of the changes that came from that proposal. It rarely makes a difference, since most proposals are small and self-contained and get implemented all-at-once in a single PR. But with bigger proposals and/or proposals with multiple features it starts to make more sense to implement them piecemeal (See my ill-considered 7k LOC PR for `ranges::to` that took 5-6 weeks to review for a good negative example). Given that we have no mechanism more fine-grained than a feature-test macro, we have no programmatic mechanism to indicate partial implementation.